### PR TITLE
orders last longer

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
reopening of #7963
orders last way longer
10->15 ticks
may increase cd later if it's too opressive but i think 50 seconds is a good time for a order duration

## Why It's Good For The Game
orders are kinda trash and I tihnk making them last longer is a good start to up their viablity

## Changelog
:cl:
balance: marine orders last 30 seconds longer
/:cl:

### CODE REVIEW

**Comment by Passe2142** [comment]
Maybe 15?. Some marines can outrun queens with 20s. 
The game is attrition based for xenos , you can have 2 SLs spamming move order for masive chase and that would mean speed like no stam pr days on a lower scale. SLs arent the only ones with move order.
Well being fair it will take to coordinate the orders. Even with 20ish of duration it not "spammable", Its hard to make ungas stay with SLs unless you point and yell.

File: code/game/objects/machinery/overwatch.dm [hunk 1]
```diff
@@ -693,8 +693,8 @@ GLOBAL_LIST_EMPTY(active_cas_targets)
 
 	if(!(command_aura in command_aura_allowed))
 		return
-	command_aura_cooldown = 45 //45 ticks
-	command_aura_tick = 10 //10 ticks
+	command_aura_cooldown = 45 //40 ticks, or 90 seconds overall CD, 35 practical.
+	command_aura_tick = 25 //25 ticks, or 50 seconds apprx.
```
Comment (BraveMole) [inline code/game/objects/machinery/overwatch.dm:697]: Lower this please, 15 seems ok

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/game/objects/machinery/overwatch.dm (hunk 1 above)
Comment (Pariah919) [reply code/game/objects/machinery/overwatch.dm:697]: ok

Comment (Pariah919) [inline code/game/objects/machinery/overwatch.dm:697]:
```suggestion
	command_aura_tick = 15//15 ticks, or 30 seconds apprx.
```

File: code/game/objects/machinery/overwatch.dm [hunk 2]
```diff
@@ -693,8 +693,8 @@ GLOBAL_LIST_EMPTY(active_cas_targets)
 
 	if(!(command_aura in command_aura_allowed))
 		return
-	command_aura_cooldown = 45 //45 ticks
-	command_aura_tick = 10 //10 ticks
+	command_aura_cooldown = 45 //40 ticks, or 90 seconds overall CD, 35 practical.
```
Comment (Pariah919) [inline code/game/objects/machinery/overwatch.dm:696]:
```suggestion
	command_aura_cooldown = 45 //40 ticks, or 90 seconds overall CD, 60 practical.
```

**Review by Pariah919 — COMMENTED** [review]
brave

**Review by BraveMole — APPROVED** [review]
Fix CL

**Pull request merged**